### PR TITLE
Add missing values materials, first

### PR DESCRIPTION
**Pull request by brandeism**
Missing Data section: A few learning objectives, lecture notes and exercises added

### CODE REVIEW

File: novice-goals.Rmd [hunk 1]
```diff
@@ -102,6 +102,46 @@ but nothing more.
...
+    -   Many descriptive statistics methods have skipna option to control if missing data should be excluded . This value is set to True by default (unlike R) 
+
+    Python Lecture Notes (useful commands)
+    -   Table: df.method() / description
+    -   dropna() / drop missing observations
+    -   dropna(how='all') / drop observations where all cells are NA
+    -   dropna(axis=1,how='all') / drop coloumn if all values are missing
+    -   dropna(thresh=5) / drop rows that contain less than 5 non-missing values
+    -   fillna(0) / replace missing values with zeros
+    -   isnull() returns True if the value is missing
```
Comment (joelostblom) [inline novice-goals.Rmd]: For consistency with `fillna()` and `dropna()`, I suggest we use the alias `isna()` instead.

File: novice-goals.Rmd [hunk 2]
```diff
@@ -102,6 +102,46 @@ but nothing more.
...
+    -   isnotnull() / Returns True for non-missing values
```
Comment (joelostblom) [inline novice-goals.Rmd]: This is without the "is"-part, and similarly to above, I suggest we use `notna()` instead.

File: novice-goals.Rmd [hunk 3]
```diff
@@ -102,6 +102,46 @@ but nothing more.
...
+
+    T/F Exercises (#Review4Everybody)
+    -   Imputation is really just making up data to artificially inflate results.  It’s better to just drop cases with missing data than to impute.
```
Comment (joelostblom) [inline novice-goals.Rmd]: I think there are some situations, such as timeseries, where imputations are still handy, especially when observations need to have the same periodicity and can't contains NaNs for downstream analysis. I agree with you below that a brief stat intro might be needed to fully grasp these concepts.

File: novice-goals.Rmd [hunk 4]
```diff
@@ -102,6 +102,46 @@ but nothing more.
...
+    Python Lecture Notes (general info) 
+    -   Missing Completely at Random (MCAR)
+    -   Missing at Random (MAR)
+    -   Missing Not at Random (MNAR)
+    -   Missing data at scale (what's a little and what's alot?)
+    -   Imputation and Multiple Imputation 
+    -   When summing the data, missing values will be treated as zero
+    -   If all values are missing, the sum will be equal to NaN
+    -   cumsum() and cumprod() methods ignore missing values but preserve them in the resulting arrays
+    -   Missing values in GroupBy method are excluded (just like in R)
```
Comment (joelostblom) [inline novice-goals.Rmd]: Potentially worth mentioning that using `.groupby()` with [`observed=True` includes NaNs for categoricals](https://pandas.pydata.org/pandas-docs/stable/whatsnew/v0.23.0.html#new-observed-keyword-for-excluding-unobserved-categories-in-groupby).

File: novice-goals.Rmd (hunk 1 above)
Comment (brandeism) [reply novice-goals.Rmd]: Sounds good, @joelostblom. I'll revise

File: novice-goals.Rmd (hunk 4 above)
Comment (brandeism) [reply novice-goals.Rmd]: Yeah, I contemplated adding .groupby() too. If it's not included elsewhere, it'll be added here.

**Comment by ChristinaLK** [comment]
No specific comments except to agree w/ the discussion that the stats intro (or at least part of it?) should go before the missing data section for context *and* I like the separation of general principles from python how-tos -- it makes sense that you provide the context for what you're doing (this and that about missing values) and then how to actually put it into practice using Python tools.

**Comment by gvwilson** [comment]
What's the status of this PR please?

**Comment by gvwilson** [comment]
Is this still live or has it gone stale?

**Pull request merged**